core-plugins/widgets/File-batchsource.json

```diff
@@ -57,17 +57,6 @@
       "label": "Output Schema Properties",
       "properties": [
         {
-          "widget-type": "textbox",
-          "label": "Path Field",
-          "name": "pathField",
-          "plugin-function": {
```

Comment:
you shouldn't be remove the entire path field. Just remove this plugin-function section.